spec.bs

```diff
@@ -2800,6 +2803,143 @@ This specification defines two [=policy-controlled features=] identified by the
 Issue(WICG/turtledove#522): Move from "`*`" to "`self`".
 
 
+# Handling Direct from Seller Signals # {#handling-direct-from-seller-signals}
+
+This section specifies a manner by which signals may be provided to auctions such that the signals
+are only used within their intended auction.
+
+Scripts running on the page may peform a `fetch()` call using the `{adAuctionHeaders: true}`
+option, which causes the value of the `Ad-Auction-Signals` response header on that request to

```

Comment:
Let's define this header _somewhere_ (I'll let you decide where it makes the most sense to define it), like we're doing with https://wicg.github.io/turtledove/#http-headerdef-x-protected-audience-bidding-signals-format-version, so we can non-normatively describe what it is to be used for.

Then we can link to that header here, to give readers more context.

Reply:
Done, I also made one for Sec-Ad-Auction-Fetch.